apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -36,6 +40,15 @@ export const asyncLoadAssessments = (sectionId, scriptId, onComplete) => {
       }
       dispatch(finishLoadingAssessments());
     });
+
+    loadAssessmentsStructureFromServer(scriptId, (error, data) => {
+      if (error) {
+        console.error(error);
+      } else {
+        dispatch(setAssessmentsStructure(scriptId, data));
+        onComplete();
+      }
+    });
```

Comment:
no need for `asyncLoadAssessments` to take an onComplete param -- you can just return the result of `loadAssessmentsStructureFromServer`, which is a jquery deferred that also implements the Promise interface